src/DssSpell.t.sol

```diff
@@ -1111,4 +1120,52 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(MedianAbstract(UNIUSD_MED).bud(SET_UNI), 1);
     }
 
+    event Log(string message, address deployer, bytes32 contractName);
```

Comment:
Check out lines 128-130. We already have some `Debug` events there, can you move this up to there?

Reply:
done